src/Form/FilePersisterServiceSettingsForm.php

```diff
@@ -471,15 +426,14 @@ public function validateForm(array &$form, FormStateInterface $form_state) {
         $valid = TRUE;
       }
       else {
-        switch ($scheme) {
-          case 's3':
-            $valid = \Drupal::service('strawberryfield.utility')
-              ->s3FilePathIsValid($path);
-            break;
-          default:
-            $valid = \Drupal::service('strawberryfield.utility')
-              ->internalFilePathIsValid($scheme, $path);
-            break;
+        // Validate for known schemes.
```

Comment:
@patdunlavey this looks much better. I will leave this at it is but will refactor tonight a bit. Why?

We have this here:
 $scheme_options = OcflHelper::getVisibleStreamWrappers();

If a scheme is not in that list, then it is actually invalid and we need to validate as "false". No worries. It works the way you wrote but I feel its good we reuse code.

Reply:
Makes sense. Thanks!